Tools/cases_generator/stack.py

```diff
@@ -0,0 +1,82 @@
+import sys
+from analyzer import StackItem
+from dataclasses import dataclass
+from formatting import maybe_parenthesize
+
+
+def var_size(var: StackItem) -> str:
+    if var.condition:
+        # Special case simplification
+        if var.condition == "oparg & 1" and var.size == "1":
+            return f"({var.condition})"
+        else:
+            return f"(({var.condition}) ? {var.size} : 0)"
+    else:
+        return var.size
+
+
+@dataclass
```

Comment:
This dataclass appears to have 0 fields, since there are no annotations in the class body. Does it need to be a dataclass at all?

```suggestion
```